apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -6620,11 +6620,11 @@
       "7d": "7D",
       "30d": "30D",
       "1y": "1Y",
-      "1H_label" : "1H",
-      "1D_label" : "1D",
-      "1W_label" : "1W",
-      "1M_label" : "1M",
-      "1Y_label" : "1Y"
+      "1H_label": "1H",
```

Comment:
Why not reusing the `1h` and `1y` above ?
You could just extend the range object and follow the same nomenclature by adding the following keys : `1d`, `1w` and `1m`